tests/test_nearest.py

```diff
@@ -1,15 +1,16 @@
 from . import unittest
+import pytest
 
 from shapely.geometry import Point
-from shapely.geos import geos_version
 from shapely.ops import nearest_points
 
-@unittest.skipIf(geos_version < (3, 4, 0), 'GEOS 3.4.0 required')
+
 class Nearest(unittest.TestCase):
     def test_nearest(self):
         first, second = nearest_points(
-                        Point(0, 0).buffer(1.0), Point(3, 0).buffer(1.0))
-        self.assertAlmostEqual(first.x, 1.0, 7)
-        self.assertAlmostEqual(second.x, 2.0, 7)
-        self.assertAlmostEqual(first.y, 0.0, 7)
-        self.assertAlmostEqual(second.y, 0.0, 7)
+            Point(0, 0).buffer(1.0), Point(3, 0).buffer(1.0),
+        )
+        assert first.x == pytest.approx(1.0, 7)
+        assert second.x == pytest.approx(2.0, 7)
+        assert first.y == pytest.approx(0.0, 7)
+        assert second.y == pytest.approx(0.0, 7)
```

Comment:
the second parameter for `pytest.approx` is not the same for `assertAlmostEqual`. Try removing it:
```suggestion
        assert first.x == pytest.approx(1.0)
        assert second.x == pytest.approx(2.0)
        assert first.y == pytest.approx(0.0)
        assert second.y == pytest.approx(0.0)
```
if that is causing issues, then I believe an equivalent is `pytest.approx(1.0, abs=1e-7)`